### ISSUE TITLE
unable to set numerical value for departPosLat

### ISSUE THREAD

**Issue by LittleWhiteRushRushRush** [eclipse/sumo#11694]
Hello, can the following situations be realized in SUMO?

Suppose a car is driving in a dual lane (with a dashed line in the middle), can it be realized that the car is driving in the middle of the two lanes (that is, the car is driving symmetrically in the center of the dashed line), rather than in a particular lane?

sumo 1.14.1

windows

**Comment by namdre** [comment]
Yes. Activate the sublanemodel and configure `<vType latAlignment="arbitrary .../>  <vehicle departPosLat="-1.6" .../>` see https://sumo.dlr.de/docs/Simulation/SublaneModel.html

**Comment by LittleWhiteRushRushRush** [comment]
Thank you very much!

I succeeded in your way. But I found a bug. I changed the value of departPosLat in the XML file. When I loaded the ROU file into the NET file, I found that the value was still 0, and I couldn't change the value through SUMO-GUI (except strings, which can't be changed to floating point numbers), as shown in the figure below.

I then tried to change the value by traci and found that although the value was modified, it did not work. My intention was to start the simulation with the car in the middle of the two lanes, but I wanted to modify his behavior by traci to make him go back to the single lane, but this seemed impossible. Sorry to bother you!

I tried it in the zip file below

![image](https://user-images.githubusercontent.com/81957538/192293404-0955d28c-5521-4906-ab56-a1d85279f063.png)
[11.Along.zip](https://github.com/eclipse/sumo/files/9646836/11.Along.zip)
